python/pyspark/sql/column.py

```diff
@@ -174,16 +175,48 @@ def _bin_op(
     ["Column", Union["Column", "LiteralType", "DecimalLiteral", "DateTimeLiteral"]], "Column"
 ]:
     """Create a method for given binary operator"""
+    binary_operator_map = {
+        "plus": "+",
+        "minus": "-",
+        "divide": "/",
+        "multiply": "*",
+        "mod": "%",
+        "equalTo": "=",
+        "lt": "<",
+        "leq": "<=",
+        "geq": ">=",
+        "gt": ">",
+        "eqNullSafe": "<=>",
+        "bitwiseOR": "|",
+        "bitwiseAND": "&",
+        "bitwiseXOR": "^",
+        # Just following JVM rule even if the names of source and target are the same.
+        "and": "and",
+        "or": "or",
+    }
 
     def _(
         self: "Column",
         other: Union["Column", "LiteralType", "DecimalLiteral", "DateTimeLiteral"],
     ) -> "Column":
+        logging_info = {}
+        if name in binary_operator_map:
+            stack = inspect.stack()
+            frame_info = stack[-1]
```

Comment:
we should respect the conf `SQLConf.get.stackTracesInDataFrameContext` and get more than one stacks if users want.

Reply:
Sure, let me address it

Reply:
Now we respect the conf.